seed/candidates.js

```diff
@@ -0,0 +1,19 @@
+conn = new Mongo();
+db = conn.getDB("ethics-api");
+
+for(var i = 1; i <= 3; i ++) {
+  var json = {
+    nameId: i,
```

Comment:
The specs for ```/api/v1/candidates/:id``` mentions the usage of NameID. See below.
![screen shot 2015-03-03 at 7 31 18 am](https://cloud.githubusercontent.com/assets/1105167/6462482/90058bce-c177-11e4-9289-7488cf0238a8.png)
I can get rid of it if this is not needed.